src/index.js

```diff
@@ -2,15 +2,14 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import ProjectionGridCore from 'projection-grid-core';
 import _ from 'underscore';
-import DefaultCell from './components/default-cell';
-import DefaultHeader from './components/default-header';
+import Defaults from './projections/default-content';
 import { TableRender } from './components/table-renderer';
 
 /* eslint-disable react/no-unused-prop-types */
 
 class ProjectionGridReact extends React.Component {
   componentWillMount() {
-    this.core = new ProjectionGridCore({ DefaultCell, DefaultHeader });
+    this.core = ProjectionGridCore.createDefault(Defaults);
```

Comment:
As I suggested in the other PR, it's better to inject the "defaults" by projection-grid-react itself.
Refer to the vue version.
https://github.com/wewei/projection-grid-vue/blob/refactoring/src/projection-grid.vue#L12

Reply:
built-in projections should be the private member of the core. React/Vue/other decorators should not have the knowledge about the order and the inner logic.

Reply:
We should avoid the duplicates. In fact the only diverged thing is the content.